server/index.js

```diff
@@ -20,13 +20,44 @@ const main = async () => {
     database: 'todo-app',
   })
   await client.connect()
+  
+  //Retrieve all tasks in a week based on the start and end date
+  app.get('/', async (req, res) => {
```

Comment:
Can you put the endpoint under the `/tasks` url?

```suggestion
  app.get('/tasks', async (req, res) => {
```